cmd/verification/main.go

```diff
@@ -48,7 +48,8 @@ const (
 	// at `requestInterval` milliseconds for each of the missing resources.
 	// When it reaches the threshold ingest engine makes a missing challenge for the resources.
 	// this value is set following this issue (3443)
-	failureThreshold = 2
+	// such a big number means we will retry forever (100 days)
+	failureThreshold = 10000000
```

Comment:
I'd like to emphasis the impact of this change again.

With a bigger retry attempts, it also possible that VN will suddenly receive lots of chunk data pack. Let's say 100. That would be ok, because the match engine uses `e.unit.Do` to verify the chunk when receiving chunk data pack in a _blocking_ manner. And since the networking priority queue will only have 5 concurrent workers, then up to 5 verifiers will work concurrently.

We had this value as a high value before, but changed it back to `2`. because high value would 



The match engine currently process chunk data pack responses in a non-blocking manner.